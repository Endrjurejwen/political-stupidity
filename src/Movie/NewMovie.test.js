import React from 'react';
import { render, fireEvent, cleanup } from 'react-testing-library';
import NewMovie from './NewMovie';

afterEach(cleanup);

test('<NewMovie', () => {
  const { getByTestId, queryByTestId, container } = render(<NewMovie />);
  const title = getByTestId('page-title');
  const movieForm = queryByTestId('movie-form');

  expect(title.textContent).toBe('New Movie');
  expect(movieForm).toBeTruthy();
  expect(container.firstChild).toMatchSnapshot();
});
