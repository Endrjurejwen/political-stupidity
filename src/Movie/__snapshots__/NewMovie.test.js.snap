// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NewMovie 1`] = `
<div>
  <h2
    data-testid="page-title"
  >
    New Movie
  </h2>
  <div>
    <form
      data-testid="movie-form"
    >
      <label
        for="text"
      >
        text
        <input
          data-testid="movie-input"
          id="text"
          type="text"
        />
      </label>
      <button
        type="submit"
      >
        Submit
      </button>
    </form>
  </div>
</div>
`;
